Lexical Scoping

```diff
@@ -0,0 +1,26 @@
+makeVector <- function(x = numeric()) {
```

Comment:
he first function, makeVector creates a special "vector", which is really a list containing a function to

set the value of the vector
get the value of the vector
set the value of the mean
get the value of the mean